swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/openapi.md

```diff
@@ -23,23 +23,26 @@ bal openapi [-i | --input] <openapi-contract-file-path>
+| `-o \                 | --output`  | The Ballerina files are generated at the same location from which the OpenAPI command is executed. Optionally, you can point to another directory location by using the optional flag `(-o\|--output)`.                                                                                                                                                                                                                                                                                                                      |
+| `--mode`              | Mode type is optional and can be either a service or client. The Ballerina service and client are generated according to the mode. Without the `--mode`, it generates both service and client stubs for the given OpenAPI contract.                                                                                                                                                                                                                                                                                      |
+| `--tags`              | To generate the Ballerina client or service stub with a subset of tags defined in the OpenAPI contract, use the `--tags` option and specify the tags you need as specified in the OpenAPI definition.<br><br>**E.g.,** `bal openapi -i <openapi-contract>  [--tags < "tag1","tag2">]`                                                                                                                                                                                                                                            |
+| `--operations`        | To generate the Ballerina client or service stub with a subset of operations defined in the OpenAPI contract, use the `--operations` option and specify the operations you need as specified in the OpenAPI definition.<br><br>**E.g.,** `bal openapi -i <openapi-contract> [--operations <"op1", "op2">]`                                                                                                                                                                                                                       |
+| `--license`           | To generate the Ballerina files with the given copyright or license header, you can use this `--license` flag with your copyright text.<br><br>**E.g.,** `bal openapi -i <openapi-contract> [--license <license-file-path>]`                                                                                                                                                                                                                                                                                                     |
```

Comment:
Can you please re-check the rendering with the these changes to the CLI reference? github rich diff shows a weird rendering the new changes made on the table

Reply:
Yes, thank you. There was an unintended formatting change. Updated the PR.